tendrl/common/manager/manager.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         self,
         name,
         integration_id,
+        node_id,
```

Comment:
Hold on, would this mean that even the gluster_integration and ceph_integration processes will know about the node_id? This isnt require

@brainfunked